neuro_extras/__init__.py

```diff
@@ -3,6 +3,7 @@
 from .main import main  # noqa
 
 
+# NOTE: When updating the version, don't forget to update main.NEURO_EXTRAS_IMAGE_TAG
```

Comment:
is this enough not to forget to bump the version of the image?

I was intended to create variable in this file, but unfortunately it's not possible as `__init__.py` depends on `main.py` not vice versa. Another option is to put version in a separate file `./NEURO_EXTRAS_IMAGE_TAG` and read it each time we run neuro-extras, but this will add too much overhead.

Reply:
you could load the version in `main.py` like `from . load NEURO_EXTRAS_VERSION`, i guess.

Reply:
no unfortunately this creates circular dependency :cry:

Reply:
ah, true

Reply:
> Another option is to put version in a separate file ./NEURO_EXTRAS_IMAGE_TAG and read it each time we run neuro-extras, but this will add too much overhead.

We use extras to ingect data once, or twice while running the jobs, shouldn't be that much overhead to read one file. Is it?

Reply:
well, I don't think it's a good practice that the work of the whole python package depends on existence of a file. Let the version be hard-coded

Reply:
or did you mean to load the file only for the operations that need it? well, that would also work but still I think too much overhead. Let's keep hard-coded version